spec/System/HapistranoSpec.hs

```diff
@@ -43,11 +43,9 @@ spec = do
            expectedOutput `Hspec.shouldSatisfy` (`isPrefixOf` actualOutput)
   describe "readScript" $
     it "performs all the necessary normalizations correctly" $ do
-#if MIN_VERSION_path_io(1,6,0)
-      let spath = $(mkRelFile "script/clean-build.sh")
```

Comment:
`spath` is going to be used in `readScript`, which has the following signature:
``` haskell
readScript :: MonadIO m => Path Abs File -> m [GenericCommand]
```

and the signature of `mkRelFile` (after lifting from the Q monad) is
``` haskell
Path Rel File
```

I'm not sure how was this being compiled in the 1st place :thinking: but seems with the last version of `path`, it now triggers an compile exception properly